.nycrc.json

```diff
@@ -5,6 +5,7 @@
     "node_modules/**",
     "coverage/**",
     "src/config/database.ts",
+    "src/migrations/*.js",
```

Comment:
why did change this file?

Reply:
When I was running tests it showed me coverage of migrations and I think it was not supposed to be there.

Reply:
That is why I added it in exclude.

Reply:
that means you moved migrations files